grpc.proxy/README.md

```diff
@@ -0,0 +1,67 @@
+# grpc.proxy
+
+Listens on https port `9090` and will echo back whatever is published to `grpc_echo` on tcp port `8080`.
+
+### Requirements
+
+- bash, jq, nc, grpcurl
+- Kubernetes (e.g. Docker Desktop with Kubernetes enabled)
+- kubectl
+- helm 3.0+
+
+### Setup
+
```

Comment:
Need to build the local docker image first, right?

Suggest enhancing `./setup.sh` to check for local `latest` image, and follow `ImagePullPolicy: Never` approach, similar to https://github.com/aklivity/zilla-examples/commit/0c5708da152bea35b05fbb4784aec7e6605146ec.